### DISCUSSION TITLE
Which CSS libraries should I use?

### DISCUSSION THREAD

**Discussion by KillerAppCreator** [community/community#83661] [Programming Help] [answered]
### Body

https://github.com/KillerAppCreator/Qustion-for-the-Community/issues/1

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by On-Fi** [comment, marked as answer]
Hey @KillerAppCreator first of all I'm also just at the beginning of my front-end journey, so take my suggestion with a grain of salt.
So with this disclaimer in place I tried a few libraries (like. Tailwind CSS, Bootstrap and Styled-Components)and I would say that [Styled-Components](https://styled-components.com/docs/basics) is easy to use.
Styled-Components use props instead of class.
Here is a [link](https://www.linkedin.com/pulse/pros-cons-using-styled-components-react-white-label-iq#:~:text=Using%20styled%2Dcomponents%2C%20you%20can,your%20website%20and%20app%20easily.&text=Styled%2DComponents%20use%20props%20instead,overall%20readability%20of%20your%20script.) for pros and cons for using Styled-Components.

All in all I would say there is no "optimal choice", it is always case-dependent which library is a good choice for your app, and which library is easy to use for yourself. So try out a few and pick one you like. :)

**Reply by KillerAppCreator** [reply]
Okay I take your suggestion

**Comment by KaveeshYoshitha** [comment]
I suggest you to use Tailwind css

**Reply by KillerAppCreator** [reply]
K

**Comment by nj-tools** [comment]
I suggest you to use Tailwind CSS.
Tailwind CSS is a utility-first CSS framework that simplifies the process of building modern and responsive user interfaces. It differs from traditional CSS frameworks like Bootstrap or Foundation by providing a set of low-level utility classes that can be directly applied to HTML elements. This approach allows for a more flexible and efficient way to design and style your web applications.
